### DISCUSSION TITLE
[chktex] 44: User Regex Warnings in tabular* environment

### DISCUSSION THREAD

**Discussion by BenjaminOddou** [James-Yu/LaTeX-Workshop#3277] [Q&A] [answered]
Hello, I am using LaTeX-Workshop v8.25.0 and I have a weird warning ⚠️ on \hline in a tabular* environment.
I have the expected output behavior. Is there a way to disable this warning ?

code :
```
\documentclass{article}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}

\begin{document}

{\renewcommand{\arraystretch}{1.5}
\noindent\begin{tabular*}{\textwidth}{c @{\extracolsep{\fill}} ccc}
\hline
column 1 & column 2 & column3 \\ \hline
I really would like this without & warnings & please \\ \hline
\end{tabular*}}

\end{document}
```

Screenshots
![vscode_bug](https://user-images.githubusercontent.com/85166574/166246010-9e75c8bf-276b-4488-9c77-a7ebb09b5ced.png)

---

![output_pdf](https://user-images.githubusercontent.com/85166574/166246263-6e6b0813-992c-400e-a11e-9347414c706a.png)

Thank you in advance for your help !

Benjamin

**Comment by jlelong** [comment]
This warning is generated by `chktex`, which is a LaTeX linter. See the [wiki](https://github.com/James-Yu/LaTeX-Workshop/wiki/Linters#chktex) for its use inside LaTeX-Workshop. See `chktex`'s documentation for how to suppress some warnings.

**Comment by BenjaminOddou** [comment, marked as answer]
Hello @jlelong and thank you for your response. I digged a little bit and found an easy fix.
Directly in the .tex file I can write `% chktex-file 44` which will mute warnings 44 for all the file. Also I can write `% chktex 44` on the problematic line to mute warnings 44 for this line only.
